### PR TITLE
ldc: update to 1.19.0

### PR DESCRIPTION
**Pull request by moon-chilled**

### CODE REVIEW

**Comment by moon-chilled** [comment]
Ahh, looks like it fails to compile on musl.  Being tracked in upstream in https://github.com/ldc-developers/ldc/issues/3211.  Marked as only for glibc for now.

File: srcpkgs/ldc/template [hunk 1]
```diff
@@ -1,24 +1,25 @@
 # Template file for 'ldc'
 pkgname=ldc
-version=1.13.0
-revision=2
+version=1.19.0
+revision=1
+archs="*-glibc"
```
Comment (xtraeme) [inline srcpkgs/ldc/template]: this is not a real fix, if the pkg was built for musl before you should make it work again.

Comment (moon-chilled) [reply srcpkgs/ldc/template]: It's not entirely clear why it doesn't work, and I don't have a musl system to test.  There is an upstream issue tracking it.

Comment (Johnnynator) [reply srcpkgs/ldc/template]: `-glibc` doesn't even exists. You want `~*-musl`

File: common/shlibs [hunk 2]
```diff
@@ -2017,11 +2017,11 @@ libFcitxQt5WidgetsAddons.so.1 libfcitx-qt5-1.2.1_1
 libfcitx-qt5.so.0 libfcitx-qt5-0.1.3_1
 liblastfm.so.1 liblastfm-1.0.9_1
 liblastfm_fingerprint.so.1 liblastfm-1.0.9_1
-libdruntime-ldc-debug-shared.so.83 ldc-runtime-1.13.0_1
-libdruntime-ldc-shared.so.83 ldc-runtime-1.13.0_1
-libphobos2-ldc-shared.so.83 ldc-runtime-1.13.0_1
-libphobos2-ldc-debug-shared.so.83 ldc-runtime-1.13.0_1
-libldc-jit.so.83 ldc-runtime-1.13.0_1
+libdruntime-ldc-debug-shared.so.89 ldc-runtime-1.19.0_1
```
Comment (Hoshpak) [inline common/shlibs]: A soname bump means that all packages that link against the library need to be revbumped to be rebuilt against the new version. In this case the packages affected are:

```
gtkd
tilix
```

According to `xrevshlib`.

**Comment by streaksu** [comment]
There is a new version of this package after 1.19.0 (1.20.1) that supposedly does not need musl patches (alpine doesn't apply anything relevant), along with this pull request going silent for months, I think one should close this one and start working on another pull request.

**Comment by ahesford** [comment]
> There is a new version of this package after 1.19.0 (1.20.1) that supposedly does not need musl patches (alpine doesn't apply anything relevant), along with this pull request going silent for months, I think one should close this one and start working on another pull request.

I'm working on the 1.20.1 update in PR #20841 but the issue is still apparent.

**Pull request closed without merging**